proposals/TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+# My New Proposal
```

Comment:
I think missing from this is sections for **Drawbacks**, **Alternatives**, and **Unresolved Questions**. Prompting the drafter to also lay out the counter argument can inspire more productive conversation.